index.html

```diff
@@ -2864,4 +2863,5 @@ <h3>*****Longer title*****</h3>
 </footer>
 </div><!-- ends "grid" -->
 </body>
+<script src="js/js.js"></script>
```

Comment:
Are you sure this should not remain inside the `<head>` rather than at the `<html>` level?

Reply:
Oh thanks, let me move it up! Or it will be invalid html.